docs/versioned/eventing/README.md

```diff
@@ -5,7 +5,7 @@ components:
 function: explanation
 ---
 
-# Knative Eventing - The Event-driven application platform for Kubernetes
+# Eventing
```

Comment:
Note that because we have explicit `.nav.yaml`, the titles here aren't reflected in the side nav, so we can have longer titles here and shorter titles in the nav.

My $0.02 here would be to keep the longer name for SEO purposes.